packages/frontend/src/sections/HowToUse.tsx

```diff
@@ -0,0 +1,103 @@
+import { ChevronRightIcon } from '@heroicons/react/24/outline';
```

Comment:
This import is causing an error.  Maybe you can import the exact same icon from react-icons.
See [this](https://react-icons.github.io/react-icons/icons/hi/)